source/includes/documentation/_vue_js_elements.md

```diff
@@ -2964,3 +2964,62 @@ Available methods in Java JDI Light:
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
 
+### 5.42 Divider
+
+[Divider Vuetify documentation page](https://vuetifyjs.com/en/components/dividers/)
+
+Divider is located in the following class:
+- __Java__: _com.epam.jdi.light.vuetify.elements.common.Divider.java_
+
+```java
+@Test
+    public void horizontalDividersTest() {
+        horizontalDividers.stream()
+                .map(Divider::is)
+                .forEach(DividerAssert::horizontal);
+    }
+
+    @Test
+    public void verticalDividersTest() {
+        verticalDividers.stream()
+                .map(Divider::is)
+                .forEach(DividerAssert::vertical);
+    }
+```
+
+__Dividers__ - The `v-divider` component is used to separate sections of lists or layouts.
+
+![Divider example](../../images/vuetify/divider.png)
+
+__Vuetify v2.6.14__ code example:
+```html
+<template>
+  <v-toolbar
+    color="purple"
+    dark
+  >
+    <v-toolbar-title>Title</v-toolbar-title>
+
+    <v-divider
+      class="mx-4"
+      vertical
+    ></v-divider>
+    </v-toolbar-items>
+
+    <v-app-bar-nav-icon></v-app-bar-nav-icon>
+  </v-toolbar>
+</template>
+
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 

![image](https://user-images.githubusercontent.com/55189259/212973943-6ebae838-34cd-45a7-9f70-17ac8e290386.png)